### ISSUE TITLE
Unable to force the indicators off on a dedicated server.

### ISSUE THREAD

**Issue by aco-rt** [diwako/diwako_dui#78]
In the cba settings of the dedicated server I have "force force diwako_dui_indicators_show = false;" yet, when one connects to the server it shows the default settings. Not sure why. The vanilla indicators are turned off.

The mod is running client side, has settings on the server, but is not running server side currently. Till now we didn't need to run it on the server.

**Comment by diwako** [comment]
That is the way CBA settings work, The server instance needs the mod or at least the CBA settings entries in order to enforce them.

DUI has support for non interface Arma instances (dedicated and HC) as it will just add the CBA settings entries and then does nothing else.

You could add it as a server mod, if you do not want it to show up on the server browser. Alternatively you could enforce these settings in your missions `cba_settings.sqf` file.

**Comment by aco-rt** [comment]
Thanks for the quick response. Let me rephrase what I said earlier, I believe I was not clear.

> Alternatively you could enforce these settings in your missions cba_settings.sqf file.

 yes, this is exactly what I have done, but on the server as opposed to the mission. 

> DUI has support for non interface Arma instances (dedicated and HC) as it will just add the CBA settings entries and then does nothing else.

This I understand, and was also the reason for not running it on the server side, as one has access to the cba settings on the server easily in the userconfig.

> In the cba settings of the dedicated server I have "force force diwako_dui_indicators_show = false;" 

Do I need to do this both on the mission and the server cba_settings.sqf ?

**Comment by diwako** [comment]
If you put these cba settings into a mission you can only do one `force` as the mission cannot overwrite or set a forced server setting.
`force diwako_dui_indicators_show = false;`

To say it again, the CBA settings config of the mod (not the ruleset that you are applying) need to be available on the server if you want it to be forced on server level ( 2x `force`). Meaning DUI needs to be loaded on the server in order for CBA to enforce the wanted rules.

If you do not want DUI to show up in the server browser load it on the server with `-serverMods` instead of `-mods`.

If you really do not want to load the mod on your server at all, force the settings via the mission `cba_settings.sqf` file, as stated in the first paragraph. This will work as the **CLIENT** has DUI loaded (thus all CBA settings configs are present) and the `cba_settings.sqf` is available to be read.

**Comment by aco-rt** [comment]
understood. Thanks for the information. :)
